content/influxdb/v2.0/reference/cli/influx/version.md

```diff
@@ -0,0 +1,23 @@
+---
+title: influx version
+description: The `influx version` command outputs the current version of the Influx CLI.
+influxdb/v2.0/tags: [influx, cli]
+menu:
+  influxdb_2_0_ref:
+    parent: influx
+weight: 202
+---
+
+The `influx version` command outputs the current version of the Influx CLI
```

Comment:
Notice we refer to this tool as "influx command line interface (CLI)" elsewhere, so let's stick with that case/spell out + period at end :-)

Reply:
we should prob update this page too https://docs.influxdata.com/influxdb/v2.0/write-data/developer-tools/influx-cli/...to be consistent across the board